qa-test-app/src/main/java/com/mapbox/navigation/qa_test_app/view/MapboxNavigationViewCustomizedActivity.kt

```diff
@@ -86,14 +92,16 @@ class MapboxNavigationViewCustomizedActivity : AppCompatActivity() {
 
         // This demonstrates that you can customize views at any time. You can also reset to
         // the default views.
-        showCustomViews.observe(this) { showCustomViews ->
+        vm.showCustomViews.observe(this) { showCustomViews ->
```

Comment:
will you please use a full name? IMO `viewModel` is acceptable but many argue that you should just spell it out

```suggestion
        customizedViewModel.showCustomViews.observe(this) { showCustomViews ->
```

`vm` is not a common abbreviation. just about every high level language style guide suggests camel case with full names. 

https://kotlinlang.org/docs/coding-conventions.html#choose-good-names
https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names
https://source.android.com/devices/architecture/hidl/code-style#naming

Reply:
Sorry if this was offensive, i do not mean it to be. I did some searching on style guides when I was wondering if i'm out of date with recent style guides. after searching, linked the findings